### PR TITLE
Make changes for v1.0 release

### PR DESCRIPTION
**Pull request by horaceko**
This PR makes the following changes in the path to v1.0.0:

* Remove deprecated properties (`clientId`, `templateKey`, `environment`, `continuation`, `role`, `email`, `phone`)
* Add a new `showCloseModalButton` property, which tells the `BerbixVerify` `iframe` to render its own close button
* Add TS type declarations

And some tooling tweaks, consisting of:

* Switching from `npm` to `yarn`

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,36 +1,20 @@
-const SDK_VERSION = "0.0.12";
+export const SDK_VERSION = "0.0.12";
```
Comment (benbeltzer) [inline src/index.js:1]: Why did we have to export it?

File: src/index.js [hunk 2]
```diff
@@ -41,66 +25,50 @@ export default {
...
       } = this;
-      if (overrideUrl != null) {
+
+      if (overrideUrl) {
         return overrideUrl;
       }
-      const token = clientToken || continuation;
-      const template = templateKey || role;
+
+      const token = clientToken;
```
Comment (benbeltzer) [inline src/index.js:49]: the extra `token` const doesn't seem necessary

File: src/index.js [hunk 3]
```diff
@@ -41,66 +25,50 @@ export default {
...
-        options.push("phone=" + encodeURIComponent(phone));
-      }
+
       if (token) {
         options.push("client_token=" + token);
       }
+
       if (showInModal) {
-        options.push("modal=true");
+        options.push(`modal=${showCloseModalButton ? 2 : 1}`);
```
Comment (benbeltzer) [inline src/index.js:57]: For understandability, could you make this an enum? (and by enum, I mean an object since this is JS?)

**Review by benbeltzer — COMMENTED** [review]
Looks good! Just a few comments.

I should note, i did not test it as I do not know how.

File: src/index.js (hunk 1 above)
Comment (horaceko) [reply src/index.js]: This was left over from when I was trying some stuff out. Thanks for catching that!

**Review by benbeltzer — APPROVED** [review]

**Review by jnnnthnn — APPROVED** [review]

**Pull request merged**